opentelemetry-sdk/src/opentelemetry/sdk/_logs/__init__.py

```diff
@@ -87,8 +87,12 @@ def to_json(self) -> str:
                 "severity_text": self.severity_text,
                 "attributes": self.attributes,
                 "timestamp": ns_to_iso_str(self.timestamp),
-                "trace_id": f"0x{format_trace_id(self.trace_id)}",
-                "span_id": f"0x{format_span_id(self.span_id)}",
+                "trace_id": f"0x{format_trace_id(self.trace_id)}"
+                if self.trace_id
+                else "",
+                "span_id": f"0x{format_span_id(self.span_id)}"
+                is self.span_id
```

Comment:
```suggestion
                if self.span_id
```

Reply:
Right, this is a bug, can a test case be added, @codeboten? :+1: